### PR TITLE
[No-28][Training]Refactor-folder-views

### PR DESCRIPTION
**Pull request by loitq**
- Structure folder resource views.

### CODE REVIEW

File: resources/views/user/blog/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                                @endif
+                            </td>
+                        </tr>
+                    @endforeach
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>
+@endsection
```
Comment (ghost) [inline resources/views/user/blog/index.blade.php]: please add newline at end of file

File: resources/views/user/sidebar.blade.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    <div class="sidebar-collapse">
+        <ul class="nav" id="side-menu">
+            <li>
+                <a href="{{ route('blog.index')}}"><i class="fa fa-edit fa-fw"></i>Blog</a>
+            </li>
+        </ul>
+        <!-- /#side-menu -->
+    </div>
+    <!-- /.sidebar-collapse -->
+</div>
```
Comment (ghost) [inline resources/views/user/sidebar.blade.php]: please add newline at end of file

File: resources/views/layouts/admin.blade.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    <script src="/asset/admin/js/plugins/dataTables/dataTables.bootstrap.js">
+    </script>
+
+    <!-- SB Admin Scripts - Include with every page -->
+    <script src="/asset/admin/js/sb-admin.js">
+    </script>
+
+</body>
+
+</html>
```
Comment (ghost) [inline resources/views/layouts/admin.blade.php]: Please add newline at end of file

File: resources/views/layouts/user.blade.php [hunk 4]
```diff
@@ -51,6 +50,7 @@
     <!-- SB Admin Scripts - Include with every page -->
     <script src="/asset/admin/js/sb-admin.js">
     </script>
+
 </body>
 
 </html>
```
Comment (ghost) [inline resources/views/layouts/user.blade.php]: please add newline at end of file

**Comment by ghost** [comment]
LGTM

**Comment by phantuan1803** [comment]
LGTM

File: app/Http/Controllers/User/BlogController.php [hunk 5]
```diff
@@ -29,9 +29,10 @@ public function index()
         }else {
             $canDelete = false;
         }
+        $canDelete = false;
```
Comment (hienl) [inline app/Http/Controllers/User/BlogController.php]: why have this line ?

File: resources/views/admin/sidebar.blade.php [hunk 6]
```diff
@@ -1,4 +1,3 @@
-<?php ?>
```
Comment (hienl) [inline resources/views/admin/sidebar.blade.php:1]: why have this removal ? why other file not have ?

File: resources/views/layouts/admin.blade.php [hunk 7]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>{{ config('app.name', 'Laravel') }}</title>
```
Comment (hienl) [inline resources/views/layouts/admin.blade.php]: This app not name 'Lavarel'

File: resources/views/layouts/user.blade.php [hunk 8]
```diff
@@ -7,7 +6,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <title>Admin</title>
+    <title>{{ config('app.name', 'Laravel') }}</title>
```
Comment (hienl) [inline resources/views/layouts/user.blade.php]: This app not name 'Lavarel'

File: resources/views/layouts/user.blade.php [hunk 9]
```diff
@@ -23,11 +22,11 @@
...
 
 <body>
-    <div id="wrapper">
-        <!-- navbar  !--->
-        @include('admin.layout.header')
-        <!-- End Header -->
-
+    
+    <!-- Page Content -->
+    <div id="wrapper">
```
Comment (hienl) [inline resources/views/layouts/user.blade.php]: why there are too many space after "wrapper"> ?

File: resources/views/partials/header.blade.php [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+    <div class="navbar-header">
+        <button type="button" class="navbar-toggle" 
+        data-toggle="collapse" data-target=".sidebar-collapse">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="{{ url('/') }}">
+            {{ config('app.name', 'Laravel') }}
```
Comment (hienl) [inline resources/views/partials/header.blade.php]: This app is not 'Lavarel'

**Review by hienl — CHANGES_REQUESTED** [review]
Please Fix

**Review by hienl — APPROVED** [review]
LGTM

**Pull request merged**